src/makeRobot.js

```diff
@@ -37,8 +38,58 @@
  *
  * @return {Robot}
  */
-function makeRobot(name, wheels, version) {
-  // write code here
+
+function makeRobot(name, wheels, chipVersion) {
+  const Robot = {
```

Comment:
```suggestion
  const robot = {
```
No need to capitalize the object name